pkg/controller/cincinnati/names.go

```diff
@@ -14,6 +14,10 @@ const (
 	NameInitContainerGraphData string = "graph-data"
 	// openshiftConfigNamespace is the name of openshift's configuration namespace
 	openshiftConfigNamespace = "openshift-config"
+	// NameConfigMapTrustedCA is the name of the copied ConfigMap from openshift-config
```

Comment:
I think this should be a suffix appended to the `Cincinnati` instance's name, so we could theoretically continue to work with more than one Cincinnati deployed in the same namespace.

Reply:
I think this goes along with https://github.com/openshift/cincinnati-operator/issues/27, which is about expanding how we discover the Cert.  Your suggestion is another path this could take, so I'll make a comment about it in the issue.  Would that suffice?

Reply:
I think in either case, if we have two Cincinnati resources in the same namespace, we don't want them trying to "share" configmaps in any way. It's just hard in that case to keep track of who is the owner, should it get cleaned up or not when a particular Cincinnati resource is deleted, etc. It's always simpler to have each primary resource have its own independent set of secondary resources, and we can't do that if there's a name collision.

Reply:
I thought you were talking about ` NameCertConfigMapKey` for some reason.  Ya, I'll change it to a suffix.

Reply:
Looking this over again, `NameConfigMapTrustedCA` is only used as the name of the Volume mounts and not the ConfigMap - I'll change the name.  The source for these volume mounts is `nameAdditionalTrustedCA()` which uses `instance.Name + "-trusted-ca"` - I'll also change the hard coded string `"-trusted-ca"` to `NameCertConfigMapKey`.  In principle, I think what I have accomplishes the ask, but I'll make the aforementioned tweeks.